pajbot/modules/chat_alerts/cheeralert.py

```diff
@@ -0,0 +1,277 @@
+                    self.get_phrase("fivethousand_bits", **payload),
+                )
+            elif num_bits >= 1500 and self.settings["fifteenhundred_bits"] != "":
+                self.bot.execute_delayed(
+                    self.settings["whisper_after"],
+                    self.bot.whisper,
+                    user,
+                    self.get_phrase("fifteenhundred_bits", **payload),
+                )
+            elif num_bits >= 500 and self.settings["fivehundred_bits"] != "":
+                self.bot.execute_delayed(
+                    self.settings["whisper_after"],
+                    self.bot.whisper,
+                    user,
+                    self.get_phrase("fivehundred_bits", **payload),
+                )
+            elif num_bits == 420 and self.settings["fourtwenty_bits"] != "":
+                self.bot.execute_delayed(
+                    self.settings["whisper_after"],
+                    self.bot.whisper,
+                    user,
+                    self.get_phrase("fourtwenty_bits", **payload),
+                )
+            elif num_bits >= 100 and self.settings["hundred_bits"] != "":
+                self.bot.execute_delayed(
+                    self.settings["whisper_after"],
+                    self.bot.whisper,
+                    user,
+                    self.get_phrase("hundred_bits", **payload),
+                )
+            elif num_bits == 69 and self.settings["sixnine_bits"] != "":
+                self.bot.execute_delayed(
+                    self.settings["whisper_after"],
+                    self.bot.whisper,
+                    user,
+                    self.get_phrase("sixnine_bits", **payload),
+                )
+            elif self.settings["one_bit"] != "":
+                self.bot.execute_delayed(
+                    self.settings["whisper_after"],
+                    self.bot.whisper,
+                    user,
+                    self.get_phrase("one_bit", **payload),
+                )
+            else:
+                log.warning("Something went wrong with the bit amount announcement!!!!")
+
+        if self.settings["grant_points_per_100_bits"] <= 0:
+            return
+
+        round_number = math.floor(num_bits / 100)
+
+        if round_number > 0:
+            points_to_grant = round_number * self.settings["grant_points_per_100_bits"]
+            user.points += points_to_grant
+            alert_message = self.settings["alert_message_points_given"]
+            if alert_message != "":
+                self.bot.say(alert_message.format(username=user, points=points_to_grant, num_bits=num_bits))
+        else:
+            return
+
+    def on_pubmsg(self, source, tags, **rest):
+        if "bits" not in tags:
+            return
+
+        if tags["bits"] != 0:
+            num_bits = int(tags["bits"])
+        else:
+            log.debug("cheeralert required bits to equal more than 0, but it is equal to 0")
+            return
```

Comment:
do a try-catch for the `ValueError` exception instead of checking the value of tags["bits"]